core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -56,6 +57,7 @@ object DiskStatus extends Enum[DiskStatus] {
   final case object Ready extends DiskStatus
   final case object Deleting extends DiskStatus
   final case object Deleted extends DiskStatus
+  final case object Error extends DiskStatus
```

Comment:
Hmm... interesting... seems like we will need to add this to the `DiskStatus` enum in terra-uii

Reply:
Yeah, for sure useful, just commenting that we need to make sure the UI knows this is a valid status (and getting typescript to tell us if we are  missing it in case statements)